apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/notes-panel.e2e.cy.ts

```diff
@@ -0,0 +1,80 @@
+describe(`open self notes panel for resource`, () => {
+    const photographBaseRoute = `/Resources/Photographs/`;
+
+    const photographId = `9b1deb4d-3b7d-4bad-9bdd-2b0d7b110002`;
+
+    describe(`when opening the self notes panel for a resource`, () => {
+        beforeEach(() => {
+            cy.visit(`${photographBaseRoute}${photographId}`);
+        });
+
+        describe(`when the photograph detail full view has loaded`, () => {
+            it(`should expose the open note panel button`, () => {
+                cy.getByDataAttribute('TextSnippetIcon').should('exist');
+            });
+
+            it(`should not expose the panel`, () => {
+                cy.get('[data-testid="notes-panel"] > .MuiPaper-root').should('not.exist');
```

Comment:
Avoid the use of MUI. Maybe look for the text for the heading of the self-notes panel instead.

Reply:
In the next refactor PR I'll move the drawer into it's own component so that this MUI specific test is isolated